src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -0,0 +1,798 @@
+				Value="{StaticResource M3MaterialButtonFontSize}" />
+		<Setter Property="FontWeight"
+				Value="Medium" />
+		<Setter Property="FontFamily"
+				Value="{StaticResource MaterialMediumFontFamily}" />
+		<Setter Property="CharacterSpacing"
+				Value="{StaticResource M3MaterialButtonCharacterSpacing}" />
+		<!--End: Label Large Typo-->
+
+		<Setter Property="MinHeight"
+				Value="{StaticResource M3MaterialButtonMinHeight}" />
+		<Setter Property="MinWidth"
+				Value="{StaticResource M3MaterialButtonMinWidth}" />
+		<Setter Property="BorderThickness"
+				Value="0" />
+		<Setter Property="Padding"
+				Value="{StaticResource M3MaterialTextButtonPadding}" />
+		<Setter Property="CornerRadius"
+				Value="{StaticResource M3MaterialButtonCornerRadius}" />
+		<Setter Property="VerticalContentAlignment"
+				Value="Center" />
+		<Setter Property="UseSystemFocusVisuals"
+				Value="True" />
+
+		<!-- Workaround for WinUI issue: https://github.com/microsoft/microsoft-ui-xaml/issues/6388 -->
+		<Setter Property="um:ControlExtensions.Icon"
+				Value="{x:Null}" />
+
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="Button">
+
+					<Grid x:Name="Root"
+						  Background="{TemplateBinding Background}"
+						  BorderBrush="{TemplateBinding BorderBrush}"
+						  BorderThickness="{TemplateBinding BorderThickness}"
+						  CornerRadius="{TemplateBinding CornerRadius}">
+
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+
+								<VisualState x:Name="Normal" />
+
+								<VisualState x:Name="PointerOver">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource PrimaryHoverBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="Pressed">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource PrimaryPressedBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="Disabled">
+									<VisualState.Setters>
+										<Setter Target="IconPresenter.Foreground"
+												Value="{StaticResource OnSurfaceDisabledBrush}" />
+										<Setter Target="ContentPresenter.Foreground"
+												Value="{StaticResource OnSurfaceDisabledBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+
+							<VisualStateGroup x:Name="FocusStates">
+
+								<VisualState x:Name="Focused">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource PrimaryFocusedBrush}" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="PointerFocused">
+									<VisualState.Setters>
+										<Setter Target="StateLayer.Background"
+												Value="{StaticResource PrimaryFocusedBrush}" />
+									</VisualState.Setters>
+								</VisualState>
```

Comment:
I believe we need to keep the PointerFocused Visual State empty to avoid the issues that this PR fixes:https://github.com/unoplatform/Uno.Themes/pull/722